cylc/rose/rose.py

```diff
@@ -59,14 +64,44 @@ def get_rose_vars(dir_=None, opts=None):
         'empy:suite.rc': None,
         'jinja2:suite.rc': None
     }
-    # Return None if dir_ does not exist
+    # Return a blank config dict if dir_ does not exist
     if not rose_config_exists(dir_):
         return config
 
-    # Load the config tree
+    # Load the raw config tree
     config_tree = rose_config_tree_loader(dir_, opts)
 
-    # For each of the template language sections...
+    # Process config_tree for environement variables in
+    # env, jinja2 and empy sections.
+    for section in ['env', 'jinja2:suite.rc', 'empy:suite.rc']:
+        # Set standardized rose-variables
+        if section not in config_tree.node.value:
+            if section != 'env':
+                continue
+            else:
+                config_tree.node.set([section])
+        config_tree.node[section].set(
+            ['ROSE_SITE'], ResourceLocator().get_conf().get_value(['site'], '')
+        )
+        config_tree.node[section].set(['ROSE_VERSION'], ROSE_VERSION)
+        config_tree.node[section].set(['CYLC_VERSION'], CYLC_VERSION)
```

Comment:
I think we will leave Cylc to handle the CYLC_VERSION var in its own codebase.

Reply:
Seems like a better idea. Happy to do that. :)